org.kde.elisa.json

```diff
@@ -0,0 +1,80 @@
+{
+    "id": "org.kde.elisa",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "elisa",
+    "finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland", "--filesystem=home:ro", "--socket=pulseaudio", "--device=dri", "--socket=session-bus", "--env=BALOO_DB_PATH=.local/share/baloo" ],
```

Comment:
Do you really need access to entire session-bus? You can get exact names by passing `--log-session-bus` to flatpak run.

Reply:
Also question is, does it work without Baloo at all?

Reply:
It should work just fine without baloo. I have changed the permissions on session bus and it works fine.
Thanks a lot for your review.